intro_series_book/02_lesson.qmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Data Munging 1"
+author: "Fay Walker"
+date: "`r format(Sys.time(), '%B %d, %Y %H:%M')`"
+output:
+  html-document:
+    number-sections: TRUE
+    self-contained: TRUE
+    code-folding: hide
+    toc: TRUE
+    toc-float: TRUE
+    css: style.css
+    editor-options:
+      chunk-output-type: console
+---
+
+```{=html}
+<style>
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
+</style>
+```
+<link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+
+# Review
+
+-   Assignment Operator
+-   Console/Environment/Script
+-   Comment your code and read your error messages
+
+# The tidyverse package
+
+-   Collection of packages using same syntax and grammar
+-   Works with "tidy" data
+-   Uses tibbles (https://r4ds.had.co.nz/tibbles.html)
+-   %\>%
+
+# Primary functions
+
+-   `select()`
+-   `rename()`
+-   `filter()`
+-   `arrange()`
+-   `mutate()`
+-   `group_by()`
+-   `summarize()`
+
+All of these functions have the same structure.
+
+1.  The first argument is the dataframe
+2.  The subsequent arguments describe what to do with the data frame, using the variable names (without quotes because you're already refering to that df)
+3.  Separate functions by %\>%
+4.  The result is a new data frame.
+
+![](C:/Users/fwalker/Downloads/image.png)
```

Comment:
This file path doesn't work!

Reply:
have no idea what the picture was supposed to be, and there wasn't a picture in the R4DS section linked so just deleted it